modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3091,6 +3108,61 @@ void ONNXImporter::parseDepthToSpace(LayerParams& layerParams, const opencv_onnx
     addLayer(layerParams, node_proto);
 }
 
+void ONNXImporter::parseIdentity(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
+{
+    layerParams.type = "Identity";
+    bool isConst = false;
+    if (layer_id.find(node_proto.input(0)) == layer_id.end())
+        isConst = true;
+
+    if (isConst)
+    {
+        std::vector<Mat> input, output;
+        input.push_back(getBlob(node_proto, 0));
+        runLayer(layerParams, input, output);
+        addConstant(node_proto.output(0), output[0]);
+        return;
+    }
+
+    addLayer(layerParams, node_proto);
+}
```

Comment:
I think we could move this code to parseSimpleLayers with minimal changed and we get support for more constant layer, what do you think?

Reply:
I would say it should be a rare case for other layers to take a constant as input. But in terms of better compatibility, let me do this.

Layers that trigger parseSimpleLayers:
https://github.com/opencv/opencv/blob/42561e1233659c30ab8f88b49212122984f6d218/modules/dnn/src/onnx/onnx_importer.cpp#L3632-L3635

Reply:
Done.